resources/docs/source/includes/_structures/user_compact.md

```diff
@@ -108,4 +108,3 @@ awarded_at  | Timestamp | |
 description | string    | |
 image_url   | string    | |
 url         | string    | |
-
```

Comment:
@nanaya is the missing final EOL going to be an issue?

Reply:
Hmm weird, looks like something caused by GitHub since I just approved the suggestion through the UI

Reply:
Oh there were originally two newlines at the end of that file for no reason (none of the other mds have it, at least), so removing one is probably better